CHANGES.md

```diff
@@ -1,5 +1,28 @@
 # Changes
 
+## edge-20.9.2
+
+This edge release continues the work of adding support for mTLS for all TCP
+traffic and changes the default container registry has been changed to
+`ghcr.io` from `gcr.io`.
+
+If you are upgrading from `stable-2.8.x` with the Linkerd CLI using the
+`linkerd upgrade` command, you must add the `--addon-overwrite` flag to ensure
+that the grafana image is properly set.
+
+* Removed the default timeout for ServiceProfiles so that ServiceProfile routes
+  behave the same as when there is no ServiceProfile definition
+* Changed default docker image repository to ghcr.io from gcr.io. **Users who
+  pull the images into private repositories should take note of this change**
+* Added endpoint labels to outbound TCP metrics to provide more context and
```

Comment:
You mentioned it above, but we should note that the proxy now load balances TCP connections, bypassing kube-proxy, and, when both endpoints are meshed, the connection is secured with mTLS. The metrics/endpoint labels are mostly in service of this.